blog/2023-02-12-conditional-export.md

```diff
@@ -0,0 +1,137 @@
+---
+slug: conditional-export
+title: 'NPM Packge: How to Conditionally Include Developer Tools?'
+authors: [sgranat]
+image: '/img/blog/free-localization/painless_localization.png'
+description: 'Creating a package with smaller bundle size for production by excluding developer tools'
+tags: [localization, i18n, translation]
+---
+
+Reducing the size of your production bundle can be a significant challenge. One common issue is including developer tools in the package, they are essential in development mode but add unnecessary bulk to the production bundle. In this article, we'll explore a solution to this problem that involves including the developer tools conditionally, based on the environment in which the package is being used.
```

Comment:
To the intro I would add some references to other similar use cases, like react query, react dev tools, redux etc. To show, that this is not just "Look how cool we are", but show that this has use cases also in outside world.

Reply:
Ok